src/components/BubbleChat/index.js

```diff
@@ -118,139 +174,108 @@ const BubbleChat = React.forwardRef(({ className, isTyping, text, type, variant,
           className && className
         )}
       >
-        <div className={classNames(
-          'BubbleChat-container',
-          'u-flex',
-          (type === 'inbound') && 'u-justifyContentEnd u-marginLeftAuto',
-        )}
-        >
-          {(avatar && type !== 'inbound') && (
-            <div className="u-flexShrink0 u-marginRightExtraSmall u-marginTopExtraSmall">
-              {
-                typeof (avatar) === 'function' ? avatar() : <Avatar name={avatar} size="small" />
-              }
-            </div>
+        <div
+          className={classNames(
+            'BubbleChat-container',
+            (type === 'inbound') && 'u-justifyContentEnd u-marginLeftAuto',
           )}
+          style={{
+            display: 'grid',
```

Comment:
Since we use `grid`, which is not supported by some browsers, we should also add `browserslist` to `package.json`

Reply:
Updated `.browserslist.rc`